third_party/kiva/cmake/compiler-flags.cmake

```diff
@@ -4,6 +4,18 @@ add_library(kiva_common_interface INTERFACE)
   # Compiler flags #
   #================#
 
+
+cmake_policy(PUSH)
```

Comment:
@nealkruis looks like some small but meaningful additions here in the Kiva build.  If you are able to just pull them into your upstream codebase, that would be great, and I'll just get this merged in.  If you have any issue with these changes, please let me know soon.